apps/projects/models.py

```diff
@@ -87,14 +88,14 @@ class Meta:
         unique_together = ("email", "project")
 
 
-class Insight(base.TimeStampedModel):
-    project = models.OneToOneField(Project, on_delete=models.CASCADE)
-    active_participants = models.PositiveIntegerField()
-    comments = models.IntegerField()
-    ratings = models.IntegerField()
-    written_ideas = models.IntegerField()
-    poll_answers = models.IntegerField()
-    live_questions = models.IntegerField()
+class ProjectInsight(base.TimeStampedModel):
+    project = models.OneToOneField(to=Project, on_delete=models.CASCADE)
+    active_participants = models.ManyToManyField(to=User)
```

Comment:
Could we also use `settings.AUTH_USER_MODEL` here like for the creator in the Invite model above? Because we set the User model in the settings.

Reply:
True, I changed it, and will add also the relation name in the OneToOne field, bcz as it is now we call project.projectinsight, and it's bit ugly and long.
@Rineee @hklarner